cfme/fixtures/v2v.py

```diff
@@ -44,23 +50,29 @@ def extract_tag(tag):
 
     if len(tags_set) < 2 or not tags_set.issubset(valid_tags):
         host_obj.remove_tags(tags=tags)
-        return True
-    return False
+        return False
+    return True
 
 
 @pytest.fixture(scope='function')
-def conversion_tags(appliance, host_creds):
+def conversion_tags(request, appliance, host_creds):
     """Assigning tags to conversation host"""
     tag1 = appliance.collections.categories.instantiate(
         display_name='V2V - Transformation Host *').collections.tags.instantiate(
         display_name='t')
+    if hasattr(request, 'param') and request.param == 'SSH':
+            transformation_method = 'SSH'
+    else:
+        transformation_method = 'VDDK'
     tag2 = appliance.collections.categories.instantiate(
         display_name='V2V - Transformation Method').collections.tags.instantiate(
-        display_name='VDDK')
-    if _tag_cleanup(host_creds, tag1, tag2):
-        host_creds.add_tags(tags=(tag1, tag2))
+        display_name=transformation_method)
+    for host in host_creds:
+        if not _tag_cleanup(host, tag1, tag2):
```

Comment:
This is a bit ambiguous to me, as the intent of _tag_cleanup's boolean return isn't clear. Please add some comments here for the intent - you're trying to add tags when the call to cleanup does not return true - which means cleanup did ... what?